crates/nao_camera/src/digital_effects.rs

```diff
@@ -0,0 +1,27 @@
+use thiserror::Error;
+
+use crate::registers::{write_register, RegisterError};
+
+#[derive(Debug, Error)]
+pub enum DigitalEffectsError {
+    #[error("failed to write register {address:#x} with value {value:#b}")]
+    RegisterNotWritten {
+        source: RegisterError,
+        address: u16,
+        value: u16,
+    },
+}
+
+pub fn disable_digital_effects(file_descriptor: i32) -> Result<(), DigitalEffectsError> {
+    let address = 0x5001;
+    let value = 0b00100011;
+    write_register(file_descriptor, 0x5001, 0b00100011).map_err(|source| {
```

Comment:
Is there a better way to document what these values mean?
In either case, why not use the values in the `write_register` call?

Reply:
I have no idea where they are coming from. I just copied them from the existing NAO camera where they are also not documented. Maybe @schmidma knows more about the values.

Nevertheless, they are now used for the `write_register` call.

Reply:
https://cdn.sparkfun.com/datasheets/Sensors/LightImaging/OV5640_datasheet.pdf page 59

Reply:
Added comments. Should be better now.